### PR TITLE
Fix bug: show vlan config for vlan with no members

### PR DESCRIPTION
**Pull request by allas-nvidia**
Signed-off-by: allas <allas@nvidia.com>

#### What I did
The problem was that there is VLAN without members `show vlan config` wasn't presented VLAN.
```
Name    VID    Member    Mode
------  -----  --------  ------
```
I modified existed code to fix this problem.
#### How I did it
File show/vlan.py, function config(db) was modified to fill the table with VLAN even without members. 
#### How to verify it
```
sudo config vlan add 10
sudo config vlan add 11
```

#### Previous command output (if the output of a command-line utility has changed)
```
Name    VID    Member    Mode
------  -----  --------  ------
```
#### New command output (if the output of a command-line utility has changed)
```
Name      VID  Member    Mode
------  -----  --------  ------
Vlan10     10
Vlan11     11
```

### CODE REVIEW

**Comment by dprital** [comment]
@allas-nvidia  - please mention which bug this PR solvs. 
What will be the new output ? printing the VLAN table also if there are no members ?

**Review by dprital — APPROVED** [review]

File: show/vlan.py [hunk 1]
```diff
@@ -110,32 +110,29 @@ def config(db):
     keys = list(data.keys())
     member_data = db.cfgdb.get_table('VLAN_MEMBER')
 
+    def member_name(member):
+        alias = member
+        if clicommon.get_interface_naming_mode() == "alias" and member is not '':
```
Comment (Junchao-Mellanox) [inline show/vlan.py]: The code 
```python
if member is not ''
```
can be changed to:
```python
if member
```

File: show/vlan.py [hunk 2]
```diff
@@ -110,32 +110,29 @@ def config(db):
...
+            alias = clicommon.InterfaceAliasConverter(db).name_to_alias(member)
+        return alias
+    
+    def member_tagging(vlan, member):
+        if member is '':
```
Comment (Junchao-Mellanox) [inline show/vlan.py]: The code 
```python
if member is '':
```

can be changed to 
```python
if not member:
```

**Review by stepanblyschak — COMMENTED** [review]
Unit test is probably required

**Review by stepanblyschak — APPROVED** [review]

**Pull request closed without merging**